services/core-web/common/components/ContactDetails.js

```diff
@@ -15,6 +15,20 @@ const fieldDefaultProps = {
   span: 12,
 };
 
+const getContactAddress = (contact) => {
+  if (!contact.address || !contact.address[0]) {
+    return "N/A";
+  }
+  const address = contact.address[0];
+  const suiteNo = address.suite_no ? `${address.suite_no}-` : "";
+  const addressLine1 = address.address_line_1 ? `${address.address_line_1}, ` : "";
```

Comment:
Just a nitpick- you should be able to null coalesce here like  `addressLine1 = address.address_line_1 ?? ""`. Little bit cleaner, but not a big deal. Or if you wanted to go all-in on optimization you could destructure at the same time. `const {suite_no="", address_line_1="",...(etc)} = address[0]`

Reply:
Thanks @taraepp this is really helpful. Will consider this in future fixes.